backend/src/main.rs

```diff
@@ -46,10 +48,16 @@ fn init_tracer(_log_file: &Option<PathBuf>, trace_level: Level) {
 }
 
 fn router(cors: CorsLayer) -> Router {
+    let mut frontend = env::current_exe().expect("Couldn't get current executable path.");
+    frontend.pop();
+    frontend.push(FRONTEND_FOLDER);
+    tracing::info!("serving frontend at: {frontend:?}");
     // Add api
     Router::new()
+        // Frontend
+        .nest_service("/", ServeDir::new(frontend))
         // Unsecured Routes
-        .route("/", get(root))
+        .route("/root", get(root))
```

Comment:
I don't see this method in `swagger-ui`. Why is that?

Reply:
the macro call was not updated when the path was changed

In the future, we need to double check whether the path defined in the `utopia` macro matches the actual path

This is the downside that should be taken into account in axum/utopia workflow.

Reply:
Maybe `ApiDoc` struct contains some info, so the check can be done in runtime with `debug_assert!`. Can you check this?

Reply:
Indeed, there is a [map](https://docs.rs/utoipa/3.1.2/utoipa/openapi/path/struct.Paths.html) with all possible paths.

in future all api calls will be `debug_assert!`'d

Reply:
Looks like you can take `OpenApi` struct ( https://docs.rs/utoipa/latest/utoipa/openapi/struct.OpenApi.html) from `ApiDoc` and check at least, that all router paths under `/api` are in `OpenApi::paths` map (checking just by name will be enough by now)

Reply:
UPDATE:
now every new API route can be added through `new_api_route`, that will check if the route in router corresponds with OpenAPI route declaration